readme.md

```diff
@@ -81,6 +81,21 @@ Default: `new nunjucks.Environment()`
 
 The custom Nunjucks [Environment object](https://mozilla.github.io/nunjucks/api.html#environment) which will be used to compile templates. If supplied, the rest of `options` will be ignored.
 
+#### filters
+
+Type: `Object`
+
+An object containing [custom filters](https://mozilla.github.io/nunjucks/api.html#custom-filters) that will be passed to Nunjucks, with the filter's name as key and the filter function as value.
+
+Example:
+
+```js
+{
+	'shorten': str => str.slice(0, 5),
+	'round': num => Math.round(num)
```

Comment:
I think we should support async filters. So when the filter function return a Promise, we would  supply the `async` flag to `env.addFilter`. That way a user could do:

```js
{
	'shorten': async str => str.slice(0, 5)
}
```

And include an example to show async support here, with a use-case that actually makes sense.

Reply:
How do you detect if a function returns a Promise?

Reply:
Actually, we don't really need to detect, we could just always make it async, by running the returned value through `Promise.resolve()` and always supply the `async` flag.